src/components/common/ForeignJurisdiction.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <div id="foreign-jurisdiction">
```

Comment:
This can still be a v-card (as you had it originally) -- any styling you put in the parent on this component will be applied to the top element in this component (inside the template block).

Reply:
I never knew that actually. OK Sev, good to know. Thanks!

Reply:
Looks good.